shell/platform/android/io/flutter/embedding/android/FlutterFragment.java

```diff
@@ -76,238 +72,268 @@
   private static final String ARG_IS_SPLASH_SCREEN_DESIRED = "show_splash_screen";
   private static final String ARG_INITIAL_ROUTE = "initial_route";
   private static final String ARG_APP_BUNDLE_PATH = "app_bundle_path";
+  private static final String ARG_DART_ENTRYPOINT = "dart_entrypoint";
+  private static final String ARG_FLUTTER_INITIALIZATION_ARGS = "initialization_args";
 
   private static final WindowManager.LayoutParams MATCH_PARENT =
       new WindowManager.LayoutParams(WindowManager.LayoutParams.MATCH_PARENT, WindowManager.LayoutParams.MATCH_PARENT);
 
+  /**
+   * Factory method that creates a new {@link FlutterFragment} with a default configuration.
+   *  - no splash screen
+   *  - initial route of "/"
+   *  - default app bundle location
+   *  - default Dart entrypoint of "main"
+   *  - no special engine arguments
+   *
```

Comment:
nit: Use Javadoc list syntax (`<li>`) so these get formatted right in the generated docs.

Reply:
done

Reply:
Can you push latest? The latest version I see doesn't include the change (here or elsewhere).

Reply:
This change appears for me in this PR.  Do you not see the <li>s under FlutterFragment?

Reply:
I don't see the change yet either.  I Javadocs, the only way to get the docs to render in a list is to use HTML tags, so this has to be:

```java
/**
 * Factory method ...
 * <ul>
 *   <li>no splash screen</li>
 *   ...
 * </ul>
 */
```

Reply:
Scratch that.  If I clicked on the code from the comment thread, GitHub very helpfully took me to the code as it existed when the first comment was added.  I have to jump through GitHub hoops to see the latest code, and yes, I see the <li> tags now 😄